test/index.test.js

```diff
@@ -386,4 +386,16 @@ describe('Fetch Tests', () => {
     const ts1 = Date.now();
     assert((ts1 - ts0) < 2000);
   });
+
+  it('fetch supports querystrings', async () => {
+    const EXPECTED = 'https://httpbin.org/delay/2?helix=dummy&foo=bar&rumple=stiltskin';
+    const qs = {
+      helix: 'dummy',
+      foo: 'bar',
+      rumple: 'stiltskin',
+    };
+    const res = await fetch('https://httpbin.org/delay/2', { qs });
```

Comment:
nitpicking again: a delayed response is not essential for this test. I'd use a different url for testing, e.g. for random bytes: 

```js
    const resp = await fetch('https://httpbin.org/bytes/512', { qs }); 
```